Experimental/DistributedAuthoritySample/Assets/Prefabs/ProBuilderMeshes/Boundary Wall 3.asset

```diff
@@ -0,0 +1,167 @@
+%YAML 1.1
```

Comment:
Just confirming from our conversation, do we need these ProBuilder files in the project?

Reply:
Yes! In this folder are the extracted meshes, which were before ProBuilderMesh Files. All these meshes are used within the scene. We can put them in a different folder though.

Reply:
Let's do that!